proposals/nnnn-binaryinteger-iseven-isodd-ismultiple.md

```diff
@@ -0,0 +1,178 @@
+# Adding `isEven`, `isOdd`, `isMultiple` to `BinaryInteger`
+
+* Proposal: [SE-NNNN](NNNN-binaryinteger-iseven-isodd.md)
+* Authors: [Robert MacEachern](https://robmaceachern.com), [SiliconUnicorn](https://forums.swift.org/u/siliconunicorn/summary)
```

Comment:
To follow the pattern used on Swift Evolution, I would like to suggest you to use the GitHub profile url address to link the authors.

e.g: 
```markdown
* Authors: [Robert MacEachern](https://github.com/robmaceachern), [SiliconUnicorn](https://forums.swift.org/u/siliconunicorn/summary)
```

Reply:
I've updated mine and have reached out to the other user to get his contact info.